launch/v4l2_camera.launch.py

```diff
@@ -126,6 +127,8 @@ def add_launch_arg(name: str, default_value=None, description=None):
                    'otherwise be RELIABLE')
     add_launch_arg('publish_rate', "-1.0",
                    description='publish frame number per second. value <= 0 means no limitation on publish rate')
+    add_launch_arg('use_v4l2_buffer_timestamps', True,
+                   description='optional flag to whether use timestamps from v4l2 buffer')
```

Comment:
```suggestion
                   description='flag to use v4l2 buffer timestamps. '
                   'If true, the image timestamps will be applied from the v4l2 buffer,  '
                   'otherwise, will be the system time when the buffer is read')
```